chore(fix): Fix Terraform Deployment Mocked Proofs for Validators and RPC Node and Fix Tests

danielntmd opened the pull request:
- Allow terraform to correctly use mocked proofs for validators and RPC node
- Removes the real proofs from 
     - `1tps.test.ts`
     - `n_tps.test.ts` 
     - `4epochs.test.ts `
     - `reorg.test.ts`
     - tests to use mocked proofs 
- Fix `transfer.test.ts` by fixing the wallet setup to correctly deploy funded accounts.

PhilWindle commented:
Nice catch. So our nodes/validators were always running with `REAL_PROOFS=true`?

Commit:
chore(fix): Allow terraform to correctly use mocked proofs for validators and RPC node
- Removes the real proofs from network tests to use mocked proofs
- Fix transfer.test.ts by fixing the wallet setup to correctly deploy funded accounts
